src/community/cog/cog-google/src/assembly/assembly.xml

```diff
@@ -0,0 +1,31 @@
+<assembly>
```

Comment:
Just a question, pom.xml has header, but assembly.xml does not? I assume it was just that way already.

I personally do not care either way as these files do not form part of the running application..

Reply:
Yeah, none of the assembly files, neither in community nor in extension, have copyright headers so far.